Assets/Plugins/Source/Editor/Utility/UnityPackageCreationUtility.cs

```diff
@@ -106,7 +112,31 @@ static UnityPackageCreationUtility()
             }
         }
 
+        public static void ReloadPackageDescriptions()
+        {
+            string pathToExportDescDirectory = Application.dataPath + "/../etc/PackageConfigurations/";
+
+            platformDescDict.Clear();
+            packagePlatformsDict.Clear();
+            isPackageExported.Clear();
+
+            var JSONPackageDescription = File.ReadAllText(pathToExportDescDirectory + "eos_platform_export_info_list.json");
```

Comment:
I'm of the opinion that string literals - particularly those referring to file names - should be stored in variables declared outside the scope of the functions in which they are used.

Reply:
This string isn't used anywhere else other than here. 
I'm under the impression that it only really matters if the file is used in multiple places.